.travis.yml

```diff
@@ -5,6 +5,7 @@ jdk:
 branches:
   only:
     - master
+    - 7226-Closed-Email-Add-Premable-for-Instructors
```

Comment:
Revert this. PRs are automatically built by Travis CI. If you want test in your own fork, do not include this change in the PR.